### PR TITLE
In Ad Landing page, images in primary content section should not scale with viewport but retain original size.

### PR DESCRIPTION
**Pull request by tatwell**
See Issue #126.

### CODE REVIEW

File: app/views/widgets/ad_landing_page/_messaging_with_image.html [hunk 1]
```diff
@@ -1,13 +1,42 @@
...
   <div class="outer-container">
     <div class="text-column">
       <div class="section-message">
-        <h3 class="title">Read what one of our graduates has to say:</h3>
+        <h3 class="title">Read what one of our graduates has to say</h3>
         <div class="body editableContent">
           <p>
-            <img class="media" src="http://www.critterbabies.com/wp-content/gallery/kittens/Cute-Kitten-kittens-16122946-1280-800.jpg"/>
-            "I came to Chapman for reason: QUALITY. Quality in education, quality in professors, quality in career services, and quality in my fellow classmates. Chapman is constantly striving for greatness, and they have given me the resources to achieve the same throughout my hourney. Because of the intimate setting Chapman provides, students are able to take full advantage of the resources and services they offer and, most importantly, get to know the faculty, staff, and fellow classmates on a more casual level."<br /><br /><strong>Jackie Becerra, MBA '15</strong><br />Nike, Inc. Intern
+            <img alt="student headshot" class="media" src="/_files/img/ad-landing-150x250.jpg">
```
Comment (tatwell) [inline app/views/widgets/ad_landing_page/_messaging_with_image.html]: Notice: if you need to include assets like images in Cascade Assets that you don't want bundled up for deployment, this is how you do it: put them in the `static` directory (see image file below).

File: app/assets/stylesheets/widgets/_ad_landing_page/messaging.scss [hunk 2]
```diff
@@ -125,7 +125,8 @@
     @include media($medium-screen) {
       display: inline-block;
       padding: 0;
-      width: 50%;
+      width: auto;
+      max-width: 300px;
```
Comment (tatwell) [inline app/assets/stylesheets/widgets/_ad_landing_page/messaging.scss]: This was the change that fixed the image stretching issue.

File: app/views/widgets/ad_landing_page/_messaging_with_image.html [hunk 3]
```diff
@@ -1,13 +1,42 @@
...
+            <img alt="large image" class="media"
+                 src="http://www.critterbabies.com/wp-content/gallery/kittens/Cute-Kitten-kittens-16122946-1280-800.jpg">
+          </p>
+          <p class="lplh-22">
+            "I came to Chapman for reason: QUALITY. Quality in education, quality in professors, quality in career services, and quality in my fellow classmates. Chapman is constantly striving for greatness, and they have given me the resources to achieve the same throughout my hourney. Because of the intimate setting Chapman provides, students are able to take full advantage of the resources and services they offer and, most importantly, get to know the faculty, staff, and fellow classmates on a more casual level."
+          </p>
+          <div class="lp-element lp-pom-text nlh" id="lp-pom-text-51">
+            <p class="p1"><strong>A Kitten Licking Its Paw</strong><br>U.S. Senator</p>
+          </div>
+          <p></p>
```
Comment (waidelacruz) [inline app/views/widgets/ad_landing_page/_messaging_with_image.html]: Empty paragraph tag.

File: app/views/widgets/ad_landing_page/_messaging_with_image.html [hunk 4]
```diff
@@ -1,13 +1,42 @@
...
-            "I came to Chapman for reason: QUALITY. Quality in education, quality in professors, quality in career services, and quality in my fellow classmates. Chapman is constantly striving for greatness, and they have given me the resources to achieve the same throughout my hourney. Because of the intimate setting Chapman provides, students are able to take full advantage of the resources and services they offer and, most importantly, get to know the faculty, staff, and fellow classmates on a more casual level."<br /><br /><strong>Jackie Becerra, MBA '15</strong><br />Nike, Inc. Intern
+            <img alt="student headshot" class="media" src="/_files/img/ad-landing-150x250.jpg">
           </p>
+          <p class="lplh-22">
+            “<i>The Schmid College faculty and staff, as well as the rigorous&nbsp;coursework, prepared me well for the challenges and&nbsp;opportunities I have encountered. Among the many extracurricular&nbsp;activities I had the opportunity to pursue as a part&nbsp;of the Ph.D. in Moon Studies, I attended the M.O.O.N. conference,&nbsp;where I landed an interview that paved the way to my dream&nbsp;job with NASA</i><i>"</i>
```
Comment (tatwell) [inline app/views/widgets/ad_landing_page/_messaging_with_image.html]: This markup was pulled directly from Cascade sample page:

http://www.chapman.edu/landing/sample-page.aspx

File: app/views/widgets/ad_landing_page/_messaging_with_image.html (hunk 3 above)
Comment (tatwell) [inline app/views/widgets/ad_landing_page/_messaging_with_image.html]: Again, a Cascade artifact. I assume it does this when a user adds an extra line to textarea field.

File: app/views/widgets/ad_landing_page/_messaging_with_image.html (hunk 4 above)
Comment (waidelacruz) [inline app/views/widgets/ad_landing_page/_messaging_with_image.html]: @tatwell Ah. I guess we can remove the second one to make consistent. Thanks!

**Pull request merged**